config/baseSchema.json

```diff
@@ -119,6 +113,37 @@
         "enableLanguages": {
             "type": "boolean",
             "default": true
+        },
+        "maxQuantity": {
+            "type": "number"
+        },
+        "twitterHandle": {
+            "type": "string"
+        },
+        "facebookAppId": {
+            "type": "string"
+        },
+        "modules": {
+            "type": "object",
+            "properties": {
+                "enabled": {
+                    "type": "object",
+                    "properties": {
+                        "shipping": {
+                            "type": "string"
+                        }
+                    },
+                    "required": ["shipping"]
+                }
+            },
+            "required": ["enabled"],
+            "default": {
+                "enabled": {
+                    "shipping": "shipping-basic"
+                }
+            },
+        "productViews": {
+            "type": "array"
```

Comment:
@mrvautin you mentioned change in settings.js and validation, isn't this part correct ?

Reply:
Not sure I haven’t had a chance to check but the Travis tests are failing. Can you run the tests and find the error?